config/resources/file-service.json

```diff
@@ -0,0 +1,51 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "nfo": "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#",
+    "dbpedia": "http://dbpedia.org/ontology/",
+    "nie": "http://www.semanticdesktop.org/ontologies/2007/01/19/nie#",
+    "dct": "http://purl.org/dc/terms/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "org": "http://www.w3.org/ns/org#",
+    "vs": "http://www.w3.org/2003/06/sw-vocab-status/ns#",
```

Comment:
Clean up those prefixes?

Reply:
Should I remove the `ext` one as well?

Reply:
Done!

Reply:
You just deleted the vs, but the other two also don't need... :)

Reply:
Time to call it a day :smile:. Sorry will fix.